hydra-node/test/Hydra/Ledger/SimpleSpec.hs

```diff
@@ -0,0 +1,40 @@
+module Hydra.Ledger.SimpleSpec where
```

Comment:
How is this related to the Commit/CollectCom changes?

Reply:
Not 100% related admittedly, this prepares follow-up work to complete the head logic. I will rebase the PR to remove the relevant commit and put it in another branch.